### DISCUSSION TITLE
Nextjs

### DISCUSSION THREAD

**Discussion by vijeyash1** [creativetimofficial/soft-ui-dashboard#43] [Q&A]
I want to use the plugin from softui dashboard pro in my nextjs. Am struggling to properly setup. Also is it possible to use the plugins like globe animation from softui pro into nextjs. I have downloaded the softui dashboard pro version of tailwind

**Comment by jjfantini** [comment]
@vijeyash1 did you ever figure something out?

**Comment by JacoboDiaz** [comment]
Could you achieve it?
